integration/tests/epochs/suite.go

```diff
@@ -127,8 +129,10 @@ func (s *Suite) Ghost() *client.GhostClient {
 	return client
 }
 
-// TimedLogf logs the message using t.Log, but prefixes the current time.
+// TimedLogf logs the message using t.Log and the suite logger, but prefixes the current time.
+// This enables viewing logs inline with Docker logs as well as other test logs.
 func (s *Suite) TimedLogf(msg string, args ...interface{}) {
+	s.log.Info().Msgf(msg, args...)
```

Comment:
do we need it?

Reply:
I found it useful because it puts the logs inline with Docker logs - makes it easier to see what's going on. You can always cross-reference by timestamp though, so not strictly necessary.